frontend/src/routes/Applications/ApplicationDetails/ApplicationTopology/topology/css/Drawer.css

```
@@ -21,6 +21,35 @@
 .topologyDetails .detailsHeader .innerDetailsHeader .detailsIconContainer {
   margin: 20px 16px 20px 20px;
 }
+
+.topologyDetails .details-view-toolbar {
+  margin-inline: var(--pf-t--global--spacer--md);
+  margin-block-end: 0;
+  --pf-v6-c-toolbar--PaddingBlockStart: var(--pf-t--global--spacer--md);
+  --pf-v6-c-toolbar--PaddingBlockEnd: var(--pf-t--global--spacer--md);
+  --pf-v6-c-toolbar--PaddingInlineStart: var(--pf-t--global--spacer--md);
+  --pf-v6-c-toolbar--PaddingInlineEnd: var(--pf-t--global--spacer--md);
+  border: 1px solid var(--pf-v5-global--BorderColor--100);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)